validationFunctions/validatePartners.js

```diff
@@ -4,17 +4,24 @@
  */
 
 /**
- * Runs custom validation on the partners array. It will be true if the supplied argument is an array having between 2 and 5 items.
+ * Runs custom validation on the partners array. It will be true if the supplied argument is an array having between 2 and 5 items with no duplicate partner values.
  *
  * @param {Object[]} partners The list of partners supplied by the user
  *
- * @returns {boolean} It will return true if the argument is an array with a minimum of 2 and a maximum of 5 items, false otherwise.
+ * @returns {boolean} It will return true if the argument is an array with a minimum of 2 and a maximum of 5 items with no duplicate partner values, false otherwise.
  */
 
-const validatePartners = partners => {
+const validatePartners = (partners) => {
   if (Array.isArray(partners)) {
-    if (partners.length >= 2 && partners.length <= 5) {
+    var uniquePartners = new Set(partners);
+    if (
+      partners.length >= 2 &&
+      partners.length <= 5 &&
+      uniquePartners.size === partners.length
+    ) {
       return true;
+    } else {
```

Comment:
This `else return false` isn't needed as it will just fall back to the outer return false statement anyway.